src/docs/variant_relative_location.md

```diff
@@ -5,18 +5,26 @@ It is important to know where in a genomic feature a variant lies and in the cas
 | Field             | Type            | Description
 |-------------------|-----------------|---------------------
 | relation          | ValueSet or null        | Type of relative location 
-| start             | int             | Start of subject within object. May be negative if upstream 
-| end               | int             | End of subject within object 
-| length            | int             | Length of overlap
+| start             | string             | Start of subject within object. May be "?" if intronic
+| end               | string             | End of subject within object.  
+| length            | int             | Length of overlap. May be negative if boundary is not known
 | percentage_overlap| float or null         | Percentage of object overlapped by the subject (Of particular interest for structural variants)
-| ref_sequence   | string          | Original sequence 
-| alt_sequence   | string          | New sequence 
+| ref_sequence   | string or null          | Original sequence 
+| alt_sequence   | string  or null        | New sequence 
 
 
 Note: 
 For `alt_sequence`, in case of frameshift variants, "X" would be easier to handle than any of the other options for now. In the future, we replace "X" with the new protein string.
 In case of deletion, `alt_sequence` will be "-" 
 
+In the case of variants with unknown start/end locations:
+For example, "628-?",  
+`start`: "628"
+`end`: "?"
```

Comment:
Probably the value of `null` here is slightly better than a magic string. Neither is great; but `null` is slightly more straightforward.

Reply:
We thought of keeping what VEP returns in the data file.

But I think what Andrey suggest is better approach. `null` sufficiently say that the end position cannot be determined. And we can keep the fields as number.